go.mod

```diff
@@ -1,27 +1,82 @@
 module github.com/aws/amazon-vpc-resource-controller-k8s
 
-go 1.16
+go 1.20
 
 require (
 	github.com/allegro/bigcache/v3 v3.1.0
 	github.com/aws/amazon-vpc-cni-k8s v1.9.0
```

Comment:
nit: while we are here, we may as well update VPC CNI dependency to latest version